MOxUnit/@MOxUnitFunctionHandleTestCase/run.m

```diff
@@ -16,6 +16,7 @@
 %
 % NNO 2015
 
+    start_tic = tic;
```

Comment:
I like the idea of measuring time - but it may be better to do this in a separate PR?
Also can the use of tic/toc be avoided? It could override any tic/toc use by the user. How about using clock() instead?